### DISCUSSION TITLE
Budibase Maximum # of Data for Table

### DISCUSSION THREAD

**Discussion by wkyoffice** [Budibase/budibase#14454] [Help]
Hello Everyone,

I am quite new to Budibase and I am facing a problem.

I have around 300K rows of data in my table and I am trying to display it using a Table in Budibase.

The thing is after 100K of data, the page started to load very slow, and when it reaches around 150K, the page won't load.

Is there any work around solution for displaying around 300K of data?

Thanks Community!

**Comment by mikesealey** [comment]
Hey @wkyoffice I think this question has come up before - the limitations come about based on the overall size of the data, rather than specific restrictions on number of rows.

Is there a reason you're trying to show 300,000 rows all at once?
